packages/plugins/rubygem-smart_proxy_acd/rubygem-smart_proxy_acd.spec

```diff
@@ -49,7 +49,7 @@ Group: Documentation
 Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
-%{?scl:Obsoletes: rubygem-%{gem_name}-doc}
+%{?scl:Obsoletes: rubygem-%{gem_name}-doc < %{version}-%{release}}
```

Comment:
It's surprising to me that this has an obsoletes for the -doc package (if SCL) but not for the main package. It feels to me that we should have both or neither. To be consistent, I'd say we should have both.

Reply:
I think it was added *after* the SCL-ization of the proxy, and for some reason we removed the non-doc, but kept the doc one…

Reply:
proof: c8588a0e8487cb2010be21974fcbc98c418294d7

Reply:
That makes sense.

Reply:
so, cleaning this up, I'd just drop the obsoletes from ACD completely? as it really not required there?

Reply:
Yes, just drop it indeed.